src/providers/AppState.js

```diff
@@ -19,15 +20,34 @@ function AppStateProvider({ children }) {
     installedApps,
     requestToken,
     stakeToken,
+    totalStaked,
+    minThresholdStakePercentage,
+    pctBase,
     ...appData
   } = useAppData(organization)
 
   const vaultBalance = useVaultBalance(installedApps, requestToken)
 
   const { balance, totalSupply } = useTokenBalances(account, stakeToken)
 
+  const effectiveSupply = useMemo(() => {
+    if (
+      !(totalSupply && totalStaked && minThresholdStakePercentage && pctBase)
+    ) {
+      return
+    }
+    const percentageOfTotalSupply = totalSupply
+      .multipliedBy(minThresholdStakePercentage)
+      .div(bigNum(1))
```

Comment:
Let's maybe create a new const in `./constants.js` namedsomething like `STAKE_PCT_BASE`.